docs: add DevTools page in v2 (#337)
vercel/swr-site: pages/docs/advanced/devtools.en-US.mdx

import Callout from 'nextra-theme-docs/callout'

# DevTools

<Callout>
  SWRDevTools is not an official project of Vercel.
</Callout>

[SWRDevTools](https://swr-devtools.vercel.app/) is a developer tool for SWR, which helps to debug your SWR cache and fetchers.

> You can install SWRDevTools on the extension pages and use it without any settings.

Reviewer: ```suggestion
You can install SWR DevTools from the extension pages and use it with zero settings!
```

- Chrome: https://chrome.google.com/webstore/detail/swr-devtools/liidbicegefhheghhjbomajjaehnjned
- Firefox: https://addons.mozilla.org/en-US/firefox/addon/swr-devtools/

After installing it, the SWR devtool panel will appear on browsers' developer tools.

> Please check the features of SWRDevTools on the website.
> 
> https://swr-devtools.vercel.app/  ([GitHub](https://github.com/koba04/swr-devtools))

Reviewer: ```suggestion
Checkout more information on the [website](https://swr-devtools.vercel.app/) and the [repository](https://github.com/koba04/swr-devtools)
```
